src/sagemaker_pytorch_container/training.py

```diff
@@ -81,12 +81,19 @@ def train(training_environment):
             runner_type = runner.PyTorchXLARunnerType
             logger.info('Invoking PT-XLA Runner')
     logger.info('Invoking user training script.')
+
+    # get capture_error from framework parameters
+    capture_error = True
+    if "sagemaker_toolkit_native_launcher_enabled" in training_environment.additional_framework_parameters:
```

Comment:
Even user specify `sagemaker_toolkit_native_launcher_enabled: False`, it will still enable the native launcher?